OSVR-Unity/CHANGES.md

```diff
@@ -5,6 +5,15 @@ This is an abbreviated changelog for the OSVR Unity Plugin.
 Use git for a full changelog.
 ##Recent Changes
 
+### InterfaceAdapter update
+> 07-July-2015 - v0.2.23 (commit c3c5dc0) and approximately v0.2.158-gb66f7af
```

Comment:
How are there two version numbers here? I'm guessing maybe one is the core? The reason the last entry had two version numbers was because we tagged v0.2 of the Unity plugin.

I get this:

```
Ryan@WINDOWS-HS7N0FC ~/Desktop/src/OSVR-Unity (master)
$ git describe --tags
v0.2-23-gc3c5dc0
```

You can just use that number (skip the stuff starting at approximately)